creator/ingest_runs/tasks.py

```diff
@@ -55,23 +55,28 @@ def check_gwo(version):
 
 @task("cancel_ingest")
 def cancel_ingest(ingest_run_uuid=None):
+    """
+    TODO - docstring
+    """
     ingest_run = IngestRun.objects.get(pk=ingest_run_uuid)
     logging.info(f"Canceling ingest run {ingest_run.pk}...")
     ingest_run.cancel()
     ingest_run.save()
 
 
 def ingest_genomic_workflow_output_manifests(ingest_run):
```

Comment:
Filled in the TBD parts to extract file content from the file version object into a DataFrame